tests/L0/run_transformer/test_pipeline_parallel_fwd_bwd.py

```diff
@@ -29,11 +29,35 @@
 
 logging.getLogger("apex").setLevel(logging.WARNING)
 
+weight_coeff = 1024
```

Comment:
[question] Is this a coeff to let the numerical check with torch.float16 as well as torch.float32?

Reply:
Yes, it is for both float16 and float32. Although, in my tests the loss based on float16 only went out of numerical bounds and became `inf` or `nan`.